tag.go

```diff
@@ -217,9 +219,6 @@ func hydrateTag(t *Tag, typeName string, isBool bool) error {
 		t.Vars[parts[0]] = parts[1]
 	}
 	t.PlaceHolder = t.Get("placeholder")
-	if t.PlaceHolder == "" {
-		t.PlaceHolder = strings.ToUpper(dashedString(typeName))
```

Comment:
Oh it makes sense now.

Reply:
Yes, I had to remove it from here, to know if placeholder is set or not, and moved it all the way there.